osu.Game.Rulesets.Osu/Objects/Drawables/DrawableSliderBall.cs

```diff
@@ -80,7 +80,7 @@ public override void ApplyTransformsAt(double time, bool propagateChildren = fal
         {
             // For the same reasons as above w.r.t rewinding, we shouldn't propagate to children here either.
             // ReSharper disable once RedundantArgumentDefaultValue - removing the "redundant" default value triggers BaseMethodCallWithDefaultParameter
-            base.ApplyTransformsAt(time, false);
+            base.ApplyTransformsAt(time);
```

Comment:
Above this line, there's a resharper disable that now looks obsolete with this change, but actually isn't because the resharper disable is broken. If there is any text after the disable that contains the substring `all`, it just disables all inspections. See also: https://github.com/ppy/osu-framework/pull/4847#issuecomment-953180147

I think it's best to revert this.

Reply:
Hmm, it is still redundant though right? Can probably jsut remove the disable and leave it be?